templates/taxi/car_list.html

```diff
@@ -1,16 +1,24 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a style="float: right" href="{% url 'taxi:car-create' %}">+</a></h1>
```

Comment:
The '+' link for creating a new car is styled with `float: right`, which might not be visually appealing or consistent across different devices and screen sizes. Consider using Bootstrap classes like `btn` and `btn-primary` for better styling and responsiveness.